docs-guides/source/opt-pruning-overview.md

```diff
@@ -41,13 +41,13 @@ be enforced while pruning, resulting in what is referred to as
 - `n:m sparsity`: When weights are divided into blocks of size `m` 
   and within each such block, `n` smallest values are chosen to be zero. 
   For example, `3:4` sparsity would imply a 75% sparse level in 
-  which 3 values in each block of size 4 are set to zero. 
+  which three values in each block of size 4 are set to ``0``. 
```

Comment:
Just out of curiosity, when should we use single backticks and to use double backticks?

Reply:
It seems double backticks are only for code formatting for .rst (restructured text) files. Since these are markdown files, I'll change all code-formatted text to single backticks.